sdk/src/lib.rs

```diff
@@ -145,15 +248,21 @@ pub use solana_inflation as inflation;
     note = "Use `solana_message::inner_instruction` instead"
 )]
 pub use solana_message::inner_instruction;
-#[deprecated(since = "2.2.0", note = "Use `solana-nonce-account` crate instead")]
-pub use solana_nonce_account as nonce_account;
+#[deprecated(since = "2.3.0", note = "Use `solana-nonce-account` crate instead")]
+pub mod nonce_account {
```

Comment:
wouldn't `solana_nonce::state` be more consistent?

Reply:
This one's also a bit annoying, it's mostly helpers for working with `solana_account` types, and not actually used by the system program implementation, except in tests.

We could potentially add it to `solana_nonce` in the future though

Reply:
ah right. these are actually my fault iirc. could deprecate them entirely in theory. maybe to `{solana_system_interface::,solana_}nonce::account` perhaps featurized to discourage future usage

Reply:
Since it's already in a separate crate, we could just stop using it downstream, then we can remove the crate entirely